third_party/move/move-compiler-v2/tests/checking-lang-v1/index.exp

```diff
@@ -0,0 +1,91 @@
+
+Diagnostics:
+error: unsupported language construct
+   ┌─ tests/checking-lang-v1/index.move:11:17
+   │
+11 │         assert!((test::R[@0x1]).value == true, 0);
+   │                 ^^^^^^^^^^^^^^^ `_[_]` index operator in non-specification code only allowed in Move 2
```

Comment:
It might be nice if we could stop giving these errors after giving a few (5, maybe).  The general idea has been received; the user must have run the wrong compiler or had the wrong idea.

But I guess that's pretty low priority.